src/Form/Validator/ErrorElement.php

```diff
@@ -186,13 +189,22 @@ public function addViolation($message, $parameters = [], $value = null)
         }
 
         $subPath = (string) $this->getCurrentPropertyPath();
+        $translationDomain = self::DEFAULT_TRANSLATION_DOMAIN;
+
+        if (\func_num_args() >= 4) {
```

Comment:
```suggestion
        // NEXT_MAJOR: Remove this hack
        if (\func_num_args() >= 4) {
```

Reply:
Can you please merge this @c-Rolland

We would like to do some major refactorings after your PR

Reply:
@core23 I can also apply the suggestion if Rhiannon’s the only thing left to do

Reply:
I tried to apply and can't

Reply:
Oh strange, me too 😌

Reply:
done ^^